### DISCUSSION TITLE
Square circle icon for macOS

### DISCUSSION THREAD

**Discussion by ViktorEvil** [squidowl/halloy#286] [General]
The icon is beautiful, but is there any chance macOS users can get a square circle version like the rest of macOS 14

**Comment by casperstorm** [comment]
Yes, in fact we have had a good conversation about it yesterday on IRC where @kozzmozz is helping.
He showed his two latest iterations which can be seen here:

<img src="https://github.com/squidowl/halloy/assets/2248455/24f16070-c982-4522-9846-8886204a1a57" width='125px'>
<img src="https://github.com/squidowl/halloy/assets/2248455/91b5a12b-73c3-4ac1-8173-69ec94dcae54" width='125px'>

Personally I think the beak is a little too close to the left edge, it feels a little too cramped.

**Comment by kozzzmozz** [comment]
Hi All,


Following to casperstorm's post above.
Let me quick introduce myself, I stumbled upon halloy recently and since then joined the irc channel.
A few days ago I was asked to tweak the current icon into a macos compatible shape which resulted in this two version.

To be clear, it's not that I'm happy about the result, but given the fact I had to start from the png under assets/ this is pretty much best I could do. I might be able to adjust it a little more to have the beak a bit more centered, but it won't be a lot.

If one could provide me with the original files, preferably layered, then I'm sure I would be able to deliver better results ... ?

Further on, let's have the discussion here going, please give me some input what you like to see changed in the icon and I will do my best to help where possible.

   1. Icon shape requests 
   2. Icon adjustment request 
   3. According to apple icon guidelines it's good to consider depicting a familiar tool to communicate what people use the app to do, we could consider adding a tool that will exceed the border to put focus on it, however it will not suit the current icon, input welcome.
  
@casperstorm you mentioned to have the bird itself exceed border, let me know if you would like me to experiment with, I think it's possible, but it seems not meeting apple guidelines to my understanding ...

And I don't know if it's appropriate, but to me it's not clear how the bird in the icon identifies itself with Halloy as an application, in case there is no particular link we could maybe consider to start from scratch ?

I hope I did not offend anyone with saying this, atleast I wrote it with my best intentions.


Cheers,
kozzmozz

**Comment by casperstorm** [comment]
For reference, here is a link to the highest resolution of the logo.

![logo](https://github.com/squidowl/halloy/assets/2248455/fef47c72-66fb-4d58-9a75-fdd10cfa87a2)

**Comment by kozzzmozz** [comment]
Thanks @casperstorm but it's not better.
Took a diferent approach and it resulted in these so far :

![halloy_reworkedv_v2_border_macos_2048](https://github.com/squidowl/halloy/assets/15728323/722cfd5f-a914-4bfc-90d8-a83d1ab39ea6)
![halloy_reworked_v2_noborder_macos_2048](https://github.com/squidowl/halloy/assets/15728323/0c703348-0d44-4707-ad6d-3218ea3bfb07)

Colors are flattened down compared to the original, just let me know what is preferred ...

Cheers!

**Reply by casperstorm** [reply]
They look fantastic. It feels like it fit much better in the square.

I like them both, but currently I think I am leaning slightly towards the one with the border 🤔

**Reply by neilalexander** [reply]
On the flip side I think the borderless one looks good, it’s not typical for macOS icons to be bordered.

**Comment by casperstorm** [comment]
#299 will resolve this discussion.
The logo will look like this:

<img width="687" alt="Screenshot 2024-03-26 at 16 45 28" src="https://github.com/squidowl/halloy/assets/2248455/19cc4621-db57-4f57-8984-8f7b1f2b3e9f">

**Reply by neilalexander** [reply]
That looks amazing!

**Reply by kozzzmozz** [reply]
This one looks better indeed, great job !
